.pytool/Plugin/SpellCheck/cspell.base.yaml

```diff
@@ -291,6 +291,7 @@
         "unenrolling",
         "unconfigure",
         "Loongson",
-        "LOONGARCH"
+        "LOONGARCH",

```

Comment:
Not a comment on this changeset, but yikes - 260 lines of unsorted words.
If someone was to sort that a) a human would now be able to check if a word is already listed and b) we wouldn't need to change the last previously added line every time, garbling history.
If you *could* fix that bit, that would be a major improvement on its own.